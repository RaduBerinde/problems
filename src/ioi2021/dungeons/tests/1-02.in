b50747e9-747c-4fca-b3b0-62317b32d2f6
10 86
1820 5250 4629 1552 6552 3205 7668 2419 6343 9299
8841 5649 9910 9479 9718 2612 7483 2360 7862 1567
8 8 9 5 6 9 8 8 9 10
4 6 4 1 4 6 8 7 4 7
4 7
3 10
4 5
2 10
2 4
0 10
3 4
3 1
0 6
0 4
5 8
9 4
4 5
5 10
7 6
6 5
1 10
8 10
6 1
4 4
9 1
8 7
6 4
5 2
5 3
4 1
5 2
5 6
6 7
8 6
7 3
2 3
6 10
9 6
3 2
3 6
5 1
5 3
2 1
2 3
7 3
4 1
9 5
4 10
7 4
3 7
4 8
7 5
2 4
9 10
6 3
7 1
4 3
8 9
6 9
6 4
1 10
7 8
8 7
3 4
9 2
9 6
4 4
1 8
8 8
3 10
8 4
0 4
6 3
4 3
3 10
5 4
1 4
2 6
8 2
6 9
1 8
2 3
9 4
4 7
3 7
2 2
7 7
0 3
6 1
9 5
