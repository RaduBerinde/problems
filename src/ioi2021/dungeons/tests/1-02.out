f39eb8f7-7d10-4b4a-af02-d7aef3d4dd0a
OK
39587
30381
39585
39782
39776
38713
30375
30372
38709
38707
25745
21992
39585
25747
22787
23130
28784
37734
23126
39584
21989
37731
23129
25739
25740
39581
25739
25743
23132
37730
22784
39775
23135
21994
30373
30377
25738
25740
39773
39775
22784
39581
21993
39590
22785
30378
39588
22786
39776
21998
23128
22782
39583
37733
23134
23129
28784
22789
37731
30375
21990
21994
39584
28782
37732
30381
37728
38707
23128
39583
30381
25741
28778
39778
37726
23134
28782
39775
21992
39587
30378
39774
22788
38706
23126
21993
